test/unit/redis_with_namespace_test.rb

```diff
@@ -8,6 +8,11 @@ def test_new_with_namespace
     assert_instance_of Redis::Namespace, redis
     assert_equal 'custom-namespace', redis.namespace
 
+    config = {'namespace': 'custom-namespace'}
```

Comment:
This is a `Symbol` No need for the quotes as it does not contain non alphanum chars